dockers/docker-config-engine-stretch/Dockerfile.j2

```diff
@@ -0,0 +1,47 @@
+FROM docker-base-stretch
+
+## Make apt-get non-interactive
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update
+
+# Dependencies for sonic-cfggen
+RUN apt-get install -y python-lxml python-yaml python-bitarray python-pip python-dev python-natsort python-setuptools
```

Comment:
python-setuptools is newly added, why?

Reply:
Without this, the installation of Python packages later in the file would fail. I think we could safely remove it during cleanup at the end of the file. I'm not quite sure what was different about the Jessie base Docker image that prevented the need for this package.